scripts/good_char.js

```diff
@@ -0,0 +1,9 @@
+/* global $ makeGoodCharRegex validCharacterPattern */
+/* exported goodChar */
+
+var goodChar;
+$(function () {
+    // need to define this after the page has loaded so validCharacterPattern
```

Comment:
should this just be at the end of the body then? Does our php js data support that?

Reply:
> Does our php js data support that?

It does not. All JS data is inserted in the header.